examples/binary_mnist_vae.py

```diff
@@ -16,7 +17,7 @@
 
 num_images = 10000
 num_pixels = num_images * 784
-batch_size = 10
```

Comment:
Changed to be comparable with the multiset codec. However, not really necessary.